src/main/java/seedu/guest/ui/GuestCard.java

```diff
@@ -54,14 +59,112 @@ public GuestCard(Guest guest, int displayedIndex) {
         this.guest = guest;
         id.setText(displayedIndex + ". ");
         name.setText(guest.getName().fullName);
+        name.setWrapText(true);
+        Tooltip nameTT = new Tooltip("Name");
+        name.setTooltip(nameTT);
+
         phone.setText(guest.getPhone().value);
+        Image phoneImage = new Image(getClass().getResourceAsStream("/images/phone.png"));
+        ImageView setPhoneImage = new ImageView(phoneImage);
+        setPhoneImage.setFitWidth(16);
+        setPhoneImage.setFitHeight(16);
+        phone.setGraphic(setPhoneImage);
+        phone.setGraphicTextGap(7);
+        phone.setWrapText(true);
+        Tooltip phoneTT = new Tooltip("Phone Number");
```

Comment:
Should we spell out TT fully as ToolTip for better clarity?

Reply:
Agreed. Changed.